dags/databricks-ml-example.py

```diff
@@ -0,0 +1,85 @@
+from datetime import datetime
+import logging
+
+from airflow.decorators import task, dag
+from airflow.providers.databricks.hooks.databricks import DatabricksHook
+from airflow.providers.databricks.operators.databricks import DatabricksSubmitRunOperator
+# from astronomer.providers.databricks.operators.databricks import DatabricksSubmitRunOperatorAsync
+
+import mlflow
+
+docs = """
+Demonstrates orchestrating ML pipelines executed on Databricks with Airflow
+"""
+
+@dag(
+    start_date=datetime(2022, 1, 1),
+    schedule_interval=None,
+    catchup=False,
+    doc_md=docs
+)
+def databricks_ml_example():
+
+    ingest_notebook = {
+        'notebook_path': '/Users/<your username>/<BigQuery-to-Databricks-notebook>',
```

Comment:
could be nice to specify the placeholders for user inputs at the top of the file, e.g.

```python
DATABRICKS_USER="<your username>"
DATABRICKS_NOTEBOOK_PATH="<path to your notebook after username>"
```